README.md

```diff
@@ -1,12 +1,14 @@
+This collection of 60+ free icons was developed by the Medic Mobile team to represent key elements of different community health workflows and protocols. While these icons were designed primarily for community health apps built with the Community Health Toolkit's [Core Framework](https://github.com/medic/medic.github.io/blob/master/resource-overviews/core-framework-overview.pdf), they can also be used in a variety of communication efforts.
 
-Use the `forms_tasks_targets` folder to find PNG and SVG icons for application code. All instances on 2.15.0 and later should be using SVG icons because they are smaller files. If your project has families or households as the lowest level of the hierarchy, use the `people_and_places` folder to swap out icons as needed so your hierarchy has the correct icons.
+<img alt="Example Icons" style="border-width:0" src="https://github.com/medic/medic.github.io/blob/master/images/icon-example-forms_tasks_targets.png?format=1000w" />
 
-NOTE: For those using medic-conf to upload resources, make sure you add the correct icons for people and places to your `resources.json` file. Otherwise, uploading resources will inadvertently remove your people and place icons. An issue is being raised in medic-conf to solve this. You'll need to define `medic-clinic`, `medic-health-center`, `medic-district-hospital` and `medic-person` in your `resources.json` and set them to the appropriate SVG files.
+The [forms_tasks_targets](https://github.com/medic/icon-library/tree/master/forms_tasks_targets) folder includes PNG and SVG icons related to diseases and conditions, treatments and proceedures, community members and health professionals, and abstract concepts like learning and health equity. The [people_and_places](https://github.com/medic/icon-library/tree/master/people_and_places) folder contains icons that address health system heirarchies, like districts, health facilities, families, and individuals.
 
-The design team has put together a resource to help with best practices when selecting and using icons in configuration. Check out the [best practices](https://docs.google.com/document/d/1vWTL9RCnMK-_jFVbQeIlIZIkfyw4DVTO-lxu6vCHieo/edit#heading=h.wlx8vh5tj0p4) to learn more about which icons can be used for which use cases and workflows.
+NOTE: We recommend using SVG icons within your health app due to their smaller size. Additionally, if you are using medic-conf to upload resources, make sure you add the correct icons for people and places to your `resources.json` file. Otherwise, uploading resources will inadvertently remove your people and place icons. An issue is being raised in medic-conf to solve this. You'll need to define `medic-clinic`, `medic-health-center`, `medic-district-hospital` and `medic-person` in your `resources.json` and set them to the appropriate SVG files.
```

Comment:
@derickl Do you know if the issue mentioned here was resolved, and if this language is still accurate?